test/conftest.py

```
@@ -137,3 +139,128 @@ def is_rtsp_stream_ready(rtsp_address):
+        port = int(port_info["HostPort"])
+
+        wait_for_port(host, port, timeout=10)
+
+        rtsp_url = f"rtsp://localhost:{port}/test"
+
+        # probe using ffmpeg to ensure the stream is ready
+        for _ in range(300):
+            try:
+                ffmpeg.probe(rtsp_url, timeout=10)
+                break
+            except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)